packages/react/src/components/form/form.stories.tsx

```diff
@@ -89,6 +92,23 @@ export const Playground: Story<FormProps<Values>> = (props) => (
       </Validation>
     </Field>
 
+    <Field>
+      <FieldLabel htmlFor="gender">
+        <span>
+          Gender <Asterisk />
+        </span>
+        <Select id="gender" name="gender" native required>
+          <Option value="">Select an option...</Option>
+          <Option value="m">Male</Option>
+          <Option value="f">Female</Option>
+          <Option value="n/a">Prefer not to answer</Option>
```

Comment:
Let's add a third option to be a bit more inclusive

```suggestion
          <Option value="n-b">Non-binary</Option>
          <Option value="n/a">Prefer not to answer</Option>
```